app/tests/components/MessageBox.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react/addons';
+import MessageBox from 'components/message-box';
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
```

Comment:
I think it doesn't necessarry to import {describe, it} from mocha, because you run them by mocha